plugins/utils/harpoon.nix

```diff
@@ -144,9 +144,16 @@ in
         Menu window height
       '';
 
-      borderChars = helpers.defaultNullOpts.mkNullable (
-        with types; listOf str
-      ) ''["─" "│" "─" "│" "╭" "╮" "╯" "╰"]'' "Border characters";
+      borderChars = helpers.defaultNullOpts.mkListOf types.str [
+        "─"
+        "│"
+        "─"
+        "│"
+        "╭"
+        "╮"
+        "╯"
+        "╰"
```

Comment:
Use [stringToCharacters](https://noogle.dev/f/lib/stringToCharacters)?

Reply:
As you want.

Reply:
Won't bother here